.circleci/config.yml

```diff
@@ -22,10 +22,10 @@ commands:
       # CircleCI breaks master branch which affects format check below. See
```

Comment:
I updated `config.yml` but is there anything else that needs to be done for `circleci` to be run correctly?

Reply:
I'm looking at the CircleCI settings ui and don't see anything related to branches, so I think we're good there!